### ISSUE TITLE
npx create-turbo@latest - strange error starting from scratch

### ISSUE THREAD

**Issue by lukeocodes** [vercel/turborepo#9013]
### Verify canary release

- [X] I verified that the issue exists in the latest Turborepo canary release.

### Link to code that reproduces this issue

Occurs when using npx create-turbo

### What package manager are you using / does the bug impact?

npm, pnpm, Yarn v1

### What operating system are you using?

Windows

### Which canary version will you have in your reproduction?

@latest (2.0.13) and @canary (2.0.13-canary.1)

### Describe the Bug

When I run `npx create-turbo@latest` on my Windows machine, I get the following error.

```bash
$ npx create-turbo@latest
? Where would you like to create your Turborepo? ./social
? Which package manager do you want to use? npm

>>> no "package.json" found at C:\Users\...\Projects\...\social
```

If I use npm to initialise the package in that directory first, I get the following error

```bash
$ npx create-turbo@latest
? Where would you like to create your Turborepo? ./social
>> social (C:\Users\...\Projects\...\social) has 1 conflicting file - please try a different location
```

To summarise;

- To a fresh path that does not exist, `create-turbo` will create the directory and then error because of no package.json found
- To an existing path, `create-turbo` will error because of no package.json
- To an existing path with a package.json, `create-turbo` will error because there is a conflicting package.json (it can only be that, it is otherwise an empty directory)

### Expected Behavior

When I run `npx create-turbo@latest` on my Windows machine, it creates a new directory set up as a turborepo

### To Reproduce

Open a new shell and run:

```bash
npx create-turbo@latest
```

Then follow the prompts you see in your terminal. Select yarn/npm/pnpm (I verified it occurs with all 3).

### Additional context

I tried this in "Command Prompt" and "git-bash" on windows

**Comment by chris-olszewski** [comment]
This should be fixed by https://github.com/vercel/turborepo/pull/9014 which is now released in `2.0.14`. 

Please use `npx create-turbo@2.0.14` as sometimes `npx create-turbo@latest` will not fetch the most recent version if it has already done it recently.
